cypress/integration/ui_spec.js

```diff
@@ -247,6 +248,115 @@ describe("UI", () => {
 				.find("[class^=error__]")
 				.should("not.exist");
 		});
+
+		it("should re-enable the input field after sending the message is successful", () => {
+			const testMessage = `Test message ${Date.now()}`;
+
+			const interception = interceptIndefinitely("POST", "*/**/messages");
+
+			visitHome();
+			clickOnLauncher();
+			sendMessage(testMessage);
+
+			// Validate that the text area is disabled
+			cy.get("@app")
+				.find("[class^=chat__]")
+				.should("be.visible")
+				.find("[class^=footer__]")
+				.should("be.visible")
+				.find("[class^=text__]")
+				.should("be.visible")
+				.find("textarea")
+				.should("be.visible")
+				.should("be.disabled")
+				.then(interception.sendResponse);
+
+			findMessage(testMessage);
+
+			// Validate that api error is visible
+			cy.get("@app")
+				.find("[class^=chat__]")
+				.should("be.visible")
+				.find("[class^=footer__]")
+				.should("be.visible")
+				.find("[class^=text__]")
+				.should("be.visible")
+				.find("textarea")
+				.should("be.visible")
+				.should("be.enabled");
+		});
+
+		it("should re-enable the input field after sending the message is unsuccessful", () => {
+			const testMessage = `Test message ${Date.now()}`;
+
+			// Make sure that posting the message fails
+			const interception = interceptIndefinitely("POST", "*/**/messages", {
+				statusCode: 400,
+				body: {
+					status: "ERROR",
+					notifications: [
+						{
+							type: "error",
+							message: "Some specific error",
+						},
+					],
+				},
+			});
+
+			visitHome();
+			clickOnLauncher();
+			sendMessage(testMessage);
+
+			// Validate that the text area is disabled
+			cy.get("@app")
+				.find("[class^=chat__]")
+				.should("be.visible")
+				.find("[class^=footer__]")
+				.should("be.visible")
+				.find("[class^=text__]")
+				.should("be.visible")
+				.find("textarea")
+				.should("be.visible")
+				.should("be.disabled")
+				.then(interception.sendResponse);
+
+			// Validate that api error is visible
```

Comment:
Je checkt niet of de error visible is, of is dat `.find("[class^=text__]")` deze?

Reply:
Scherp, er is idd geen error was van een copy paste
Done